conf/logback.xml

```diff
@@ -39,10 +26,6 @@
         <appender-ref ref="ACCESS_LOG_FILE" />
     </logger>
```

Comment:
I think this section must also be deleted.

Reply:
Good spot

Reply:
I think I might have ctrl-z'd this one by mistake